core/src/main/java/nu/yona/server/device/service/UserDeviceDto.java

```diff
@@ -138,4 +128,15 @@ public DeviceRegistrationRequestDto(@JsonProperty("name") String name,
 			this.appVersion = appVersion;
 		}
 	}
+
+	public static class DeviceUpdateRequestDto
```

Comment:
🔧 I would prefer to have this in a separate file for clarity.

Reply:
OK